src/utilities/getNumericalClasses.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Get numerical classes
+ * @param classes Array of classes as strings.
+ * @return Array of classes as numbers.
+ */
+
+export default function getNumericalClasses(labels: string[]) {
+  const categories = [...new Set(labels)].sort((a: any, b: any) => a - b);
```

Comment:
Why do we need to sort here ? 
And if we want to sort why should we put a callback and convert strings to number ?